frontend/src/components/videos/feed_video.jsx

```diff
@@ -51,7 +54,10 @@ class VideoPlayer extends React.Component {
     });
   }
 
-  saveVid() {}
+  saveVid() {
+   const body = {userId: this.props.userId}
+   this.props.savePost(this.props._id, body).then(user => console.log(user))
```

Comment:
Can we get rid of this print statement? Do you want to replace it with a different ```then``` function for some sort of verification?